src/components/AdminPage/AdminPanel/AdminPanel.js

```diff
@@ -1,5 +1,92 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useCookies } from "react-cookie";
+import styles from "./AdminPanel.module.css";
+import { auth } from "../../../firebase";
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { Button, Form, FormGroup, FormControl } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Title from "../../Title/Title";
+
 const AdminPanel = () => {
-  return <div>AdminPanel</div>;
-};
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [, setCookie] = useCookies(["isSignedIn"]);
+  const navigate = useNavigate();
+
+  const navigation = () => {
+    navigate("/admin/home");
+  };
 
+  const signInUser = () => {
+    signInWithEmailAndPassword(auth, email, password)
+      .then((res) => {
+        setCookie("isSignedIn", true, { path: "/" });
+        navigation();
+      })
+      .catch((err) => {
+        alert("Invalid e-mail or password!");
+      });
+  };
+
+  return (
+    <div className={styles.loginPage}>
+      <Form className="d-flex flex-column mw-40 mh-40 p-3 justify-content-center text-center align-items-center">
+        <img src="/icons/logo_icon.png" className="fluid" alt="" />
+        <Title />
+        <FormGroup
+          className={`d-flex flex-column ${styles.container}`}
+          style={{ minWidth: "20vw" }}
+        >
+          <FormGroup
+            className={`mb-3 text-white bg-transparent rounded mt-3 mb-3 ${styles.form}`}
+            style={{
+              height: "8vh",
+              borderColor: "#fff",
+            }}
+          >
+            <FormControl
+              className="input"
+              type="email"
+              placeholder="e-mail"
+              value={email}
+              onChange={(e) => setEmail(e.target.value)}
+              style={{
+                height: "8vh",
```

Comment:
To improve the design, please remove every height 8vh and set in the file with the styles min-height instead of height.